cfgov/jinja2/v1/es/es-base.html

```diff
@@ -1,51 +1,39 @@
-<!DOCTYPE html>
-<!--[if lt IE 7]>      <html lang="es" class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html lang="es" class="no-js lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html lang="es" class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html lang="es" class="no-js"> <!--<![endif]-->
-<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb#">
-
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title>{{ self.babel_title() | trim }}</title>
-    <meta name="description" content="">
-    <meta name="viewport" content="width=device-width initial-scale=1">
-    <meta name="title" content="{{ self.babel_title() | trim }}" />
-    <meta property="og:title" content="{{ self.babel_title() | trim }}" />
-    <meta property="og:url" content="{{ request.url }}">
-    <meta property="og:type" content="government">
-    <meta property="og:site_name" content="Oficina para la Protección Financiera del Consumidor">
-    <meta property="fb:page_id" content="141576752553390">
-    <meta property="fb:app_id" content="210516218981921">
-    <meta name="Copyright" content="Public domain">
-    <link rel="apple-touch-icon" href="{{ static('nemo/_/img/apple-touch-icon-precomposed.png') }}">
-    <link rel="apple-touch-icon" size="72x72" href="{{ static('nemo/_/img/apple-touch-icon-72x72-precomposed.png') }}">
-    <link rel="apple-touch-icon" size="114x114" href="{{ static('nemo/_/img/apple-touch-icon-114x114-precomposed.png') }}">
-    <link rel="apple-touch-icon" size="144x144" href="{{ static('nemo/_/img/apple-touch-icon-144x144-precomposed.png') }}">
+{% extends 'base-common.html' %}
+
+{%- block title -%}
+    {%- if self.babel_title() -%} 
+        {{ self.babel_title() | trim }}
+    {%- endif %}
+{%- endblock title -%}
 
+{% block og_type %}government{% endblock %}
+{% block og_site_name %}Oficina para la Protección Financiera del Consumidor{% endblock %}
 
+{% block og_page_id %}
+    <meta property="fb:page_id" content="141576752553390">
+{% endblock %}
+
+{% block modernizr %}
     <!-- Custom modernizr for touch detection and svg support only, see the .js file for the build link -->
     <script src="{{ static('nemo/_/js/modernizr.custom.js') }}"></script>
+{% endblock %}
 
+{% block css %}
     <!--[if lt IE 9]>      <link rel="stylesheet" href="{{ static('nemo/_/c/es-styles-ie.min.css') }}"> <![endif]-->
     <!--[if gt IE 8]><!--> <link rel="stylesheet" href="{{ static('nemo/_/c/es-styles.min.css') }}"> <!--<![endif]-->
 
     <!--[if lt IE 9]> <script src="{{ static('nemo/_/js/html5shim.js') }}"></script> <![endif]-->
```

Comment:
Same as earlier comment on conditional comments, are these not missing a closing HTML comment tag at the beginning?